### ISSUE TITLE
[QUESTION] how to get ironic logs?

### ISSUE THREAD

**Issue by petro-rudenko** [metal3-io/baremetal-operator#1110]
Hi, sorry for the probably silly question, found there's a [ironic-debuggability-improvement document](https://github.com/metal3-io/metal3-docs/blob/59a48aa08aa67f054a8f1455772b2a4ec9055573/design/ironic-debuggability-improvement.md) but still can't find an answer.

I've setup local ironic, but got ipmi error:
```
 During sync_power_state, max retries exceeded for node 50575b5b-3507-43b0-a8e3-1941cf53d2ad, node state Non
e does not match expected state 'power on'. Updating DB state to 'None' Switching node to maintenance mode. Error: Command '['ipmitool
', '-I', 'lanplus', '-H', '*.*.*.*', '-L', 'ADMINISTRATOR', '-U', 'ADMIN', '-C', '17', '-R', '7', '-N', '5', '-f', '/tmp/tmpn3zyj5
_l', 'power', 'status']' timed out after 60 seconds |
| management_interface   | ipmitool
```

Running this command from ironic inspector container works fine. And I can't find ironic log file neither in `/shared` folder, nor in `/var/log`
Basically, I want to find somewhere this log line: 

https://github.com/openstack/ironic/blob/53a38f18d47d3fef5e9a7a8f6ecf5ab99b2f8e0f/ironic/drivers/modules/ipmitool.py#L778
